### PR TITLE
OADP: Add jobs for 4.8 stable

### PR DESCRIPTION
**Pull request by hhpatel14**

### CODE REVIEW

**Review by weshayutin — APPROVED** [review]
Thanks Hiteshwari

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.8.yaml [hunk 1]
```diff
@@ -0,0 +1,161 @@
+base_images:
+  base:
+    name: "4.9"
```
Comment (kaovilai) [inline ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.8.yaml:3]: Why is this 4.9?

**Review by kaovilai — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.8.yaml (hunk 1 above)
Comment (hhpatel14) [reply ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.8.yaml:3]: Line 32 gives us cluster with requested version for E2E test.

Comment (kaovilai) [reply ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.8.yaml:3]: Should be fine. Just an image builder it appears.

**Review by kaovilai — APPROVED** [review]

**Comment by petr-muller** [comment]
/test ordered-prow-config

This PR needs to be regenerated using `make jobs`

**Comment by petr-muller** [comment]
/lgtm 
/hold

Holding for rehearsal, feel free to unhold if satisfied

**Pull request merged**